docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -1,26 +1,25 @@
 [id="adopting-the-object-storage-service_{context}"]
 
-= Adopting the Object Storage service
+= Adopting the {object_storage}
 
-This section only applies if you are using OpenStack Swift as {object_storage_first_ref}. If you are using the Object Storage API of Ceph RGW this section can be skipped.
+If you are using OpenStack Swift as the {object_storage_first_ref}, adopt the {object_storage} to the {rhos_long} environment. If you are using the Object Storage API of the Ceph Object Gateway (RGW), skip this procedure.
 
 .Prerequisites
 
-* Previous adoption steps completed.
-* The {object_storage} storage backend services must still be running.
-* Storage network has been properly configured on the {OpenShift} cluster.
+* Ensure that the {object_storage} storage back-end services are running.
+* Ensure that the storage network is properly configured on the {OpenShift} cluster.
```

Comment:
@cschwede How would customers ensure that their storage network is configured prooperly?
One suggestion is to link to the following guide, if applicable: https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/configuring_storage/index

Reply:
I propose to link here to [https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-rhocp-for-rhoso#proc_configuring-the-data-plane-network_preparing](https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html/deploying_red_hat_openstack_services_on_openshift/assembly_preparing-rhocp-for-rhoso#proc_configuring-the-data-plane-network_preparing)